packages/plugins/src/hardhat/tasks.ts

```diff
@@ -350,6 +350,7 @@ task(TASK_CHUGSPLASH_PROPOSE)
     'skipStorageCheck',
     "Upgrade your contract(s) without checking for storage layout compatibility. Only use this when confident that the upgrade won't lead to storage layout issues."
   )
+  .addFlag('remoteExecution', 'Upload config file to IPFS for remote execution')
```

Comment:
We actually shouldn't have `remoteExecution` as a parameter to the `chugsplashProposeTask`, since the user doesn't need to think about this. Instead of having `remoteExecution` as a parameter, we can do this:
```
const remoteExecution = (await getChainId(provider)) !== hre.config.networks.hardhat.chainId
```
We do this everywhere else in our codebase to get the value of `remoteExecution`

Reply:
So the problem with this is that it entirely prevents me from testing this feature on a local network. I see why you don't want it visible to the user, but I do think we should try to make it possible to test things locally.